src/renderer/themes.css

```diff
@@ -261,6 +271,45 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --logo-text: url("../../_icons/textNordicLightSmall.png");
 }
 
+.gruvboxDark {
+  --primary-text-color: #ebdbb2;
+  --secondary-text-color: #fbf1c7;
+  --tertiary-text-color: #ebdbb2;
+  --title-color: #ebdbb2;
+  --bg-color: #1d2021;
+  --favorite-icon-color: #fabd2f;
+  --card-bg-color: #282828;
+  --secondary-card-bg-color: #282828;
```

Comment:
**issue (blocking):** Please choose a color that has an acceptable color contrast (>=3:1) with the card background color (`--card-bg-color`). Examples of where this is used: the "About" tags on a channel, the author name outline on a comment made by the video author in a video's comments section.